langchain/src/tools/apify.ts

```diff
@@ -0,0 +1,93 @@
+import { Document } from "../document.js";
```

Comment:
Tools have a specific meaning in LangChain - this is in the wrong place. Can we bundle it into the document loader and run `callActor` when `load`'ing documents?

Reply:
Thanks for the review.
In the Python version, it's in the utilities. https://github.com/hwchase17/langchain/blob/master/langchain/utilities/apify.py
Not sure if there's an equivalent of it in the JS version?

Reply:
In that case I'd prefer to put this as a `callActor` method in the `document_loader` class. I can make the change today and polish everything up if that's ok with you?

Reply:
The other reason is so that we can give everything that requires Apify one entrypoint.

Reply:
Thank you @jacoblee93 If you prefer so, feel free to update it that way.